pkg/kubelet/network/plugins.go

```diff
@@ -215,6 +216,9 @@ func (plugin *NoopNetworkPlugin) Init(host Host, hairpinMode kubeletconfig.Hairp
 	if err := utilsysctl.New().SetSysctl(sysctlBridgeCallIPTables, 1); err != nil {
 		glog.Warningf("can't set sysctl %s: %v", sysctlBridgeCallIPTables, err)
 	}
+	if err := utilsysctl.New().SetSysctl(sysctlBridgeCallIP6Tables, 1); err != nil {
+		glog.Warningf("can't set sysctl %s: %v", sysctlBridgeCallIP6Tables, err)
```

Comment:
Does this sysctl even exist if the kernel is booted with IPv6 disabled or compiled out?  Not sure if we care about that case, but people still do this when they don't want IPv6, and I believe those make the sysctl simply not present and thus will always error spuriously.

Perhaps you can check if the err is os.IsNotExist() and ignore the warning?

Reply:
I can ignore the warning when the element doesn't exist. There is a GetSysctl() to see if it exists, I will look at using that.